lib/integration/esbuild.js

```diff
@@ -97,7 +97,17 @@ export function esbuild(options = {}) {
     async function onloadremote(data) {
       const href = data.path
       console.log('%s: downloading `%s`', remoteNamespace, href)
-      const contents = (await got(href, {cache})).body
+
+      /** @type {string} */
+      let contents
+
+      const cachedContents = cache.get(href)
+      if (cachedContents) {
+        contents = cachedContents
+      } else {
+        contents = await (await fetch(href)).text()
+        cache.set(href, contents)
+      }
```

Comment:
This issue may have already existed before, but now is more visible.
It's a bit worrying that the caching system doesn't have memory management (max cached items, max memory usable, etc).
On massive projects, caching could become an issue exceeding the memory node is allowed.

Reply:
Yeah, not sure whether that should be improved as part of this PR though, might be better do to it in a separate PR?

Reply:
I’m not *too* worried about it. Could use something smarter in the future (an LRU cache?), but this is probably fine for now in build scripts